packages/calypso-e2e/src/lib/pages/themes-page.ts

```diff
@@ -134,4 +135,25 @@ export class ThemesPage {
 	async validateCurrentTheme( expectedTheme: string ): Promise< void > {
 		await this.page.waitForSelector( selectors.currentTheme( expectedTheme ) );
 	}
+
+	/**
+	 * Visit the Theme showcase page.
+	 *
+	 * @param siteSlug
+	 */
+	async visitShowcase( siteSlug: string | null = null ) {
+		const targetUrl = `themes/${ siteSlug ?? '' }`;
+
+		// We are getting a pending status for https://wordpress.com/cspreport intermittently
+		// which causes the login to hang on networkidle when running the tests locally.
+		// This fulfill's the route request with status 200.
+		// See https://github.com/Automattic/wp-calypso/issues/69294
+		await this.page.route( '**/cspreport', ( route ) => {
+			route.fulfill( {
+				status: 200,
+			} );
+		} );
+		await this.page.goto( getCalypsoURL( targetUrl ) );
+		await this.page.waitForLoadState( 'networkidle' );
```

Comment:
Same as above, `goto` waits for the load state as well, so this line can be removed. Also, waiting until `networkidle` is discouraged as we should rely on web assertions instead. We're probably fine with simply removing it, though :)